### PR TITLE
Refactored tests

### LINKED ISSUE #284: Refactor Tests
As a developer, I'd like to use one test call consistently (test vs. it), in order to have tests that are better to read.

---

### :question: &nbsp; Additional Questions to Answer / Consider

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Should check whether it makes more sense to use `test` or `it` and decide for one 
- [ ] Should update all tests to use the one option you decided for

### PR DESCRIPTION
**Pull request by SanJSp**
Fixes #284 

## Changes
This PR changes all test case keywords from `test`to `it` as we had more it statements than test statements. Furthermore, I've adapted the test case descriptions to have better grammar/spelling.

## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

### CODE REVIEW

**Review by lukashueller — APPROVED** [review]
Looks good for me. Thanks for also addressing the grammar mistakes and typos!

File: client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js [hunk 1]
```diff
@@ -35,15 +35,15 @@ const taskSectionAsArray = parser.getRobotCodeAsArray(correctTaskSection);
 const declaredApplications = parser.getApplicationArray(settingsSectionAsArray);
 
 describe('RobotCode to SSOT Parsing Tests', () => {
-  test('ssot contains right robotMetadata', () => {
+  it('ssot contains right robotMetadata', () => {
```
Comment (WolfgangDaniel) [inline client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js:38]: This seems to be the only "it" that is not adjusted grammatically. Maybe we change it to something like "checks if ssot contains right robotMetadata"

**Review by WolfgangDaniel — CHANGES_REQUESTED** [review]
Overall a great job. Just one small remark :)

File: client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js (hunk 1 above)
Comment (SanJSp) [reply client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js]: Your proposition is a bit tricky, as one should not refer to the test itself when describing the test. I've changed it to 
` it('contains the correct robotMetadata', () => {` as the test suite description describes what end product (the ssot) results from the test.

**Review by WolfgangDaniel — APPROVED** [review]

**Pull request merged**